src/js/api/config.js

```diff
@@ -122,6 +123,9 @@ define([
             ]);
 
             config.playlist = [ obj ];
+        } else if (_.isArray(configPlaylist.playlist)) {
+            config.feedData = configPlaylist;
+            config.playlist = configPlaylist.playlist;
```

Comment:
When you inline a feed, it looks like this:
```js
jwplayer().setup({
   playlist: {
    playlist: [
      {
        file: 'video.mp4'
      }
    ],
    title: 'Feed Title',
    pubdate: '2017-04-06'
  }
});
```
So the "playlist" in the config is actually a feed that contains a playlist.